charts/backstage/templates/backstage-deployment.yaml

```diff
@@ -66,6 +72,17 @@ spec:
           env:
             - name: APP_CONFIG_backend_listen_port
               value: {{ .Values.backstage.containerPorts.backend | quote }}
+            - name: POSTGRES_HOST
```

Comment:
here we could:

```yaml
- name: APP_CONFIG_backend_database_client
  value: pg
- name: APP_CONFIG_backend_database_connection_port
  value: {{ .Values.backstage.containerPorts.backend | quote }}
- name: APP_CONFIG_backend_database_connection_port
  value: {{ include "backstage.databasePort" . }}
- name: APP_CONFIG_backend_database_connection_user
  value: {{ include "backstage.databaseUser" . }}      
- name: APP_CONFIG_backend_database_connection_password
  valueFrom: ...
```

so that we don't depend on users setting up the proper environment variables in their `app-config.yaml`

Reply:
Yep, am fine with that, the only reason I didn't put them as overrides was because of the following section in the [Configuration docs](https://backstage.io/docs/conf/#supplying-configuration)

> However these should be used sparingly, usually just for temporary overrides during development or small tweaks to be able to reuse deployment artifacts in different environments.

But if you feel in this case it is fine, I'll make the change :+1:

Reply:
yeah good point